commands/eval.js

```diff
@@ -3,18 +3,37 @@
 // can be used to do **anything** on your machine, from stealing information to
 // purging the hard drive. DO NOT LET ANYONE ELSE USE THIS
 
-
 // However it's, like, super ultra useful for troubleshooting and doing stuff
 // you don't want to put in a command.
 exports.run = async (client, message, args, level) => { // eslint-disable-line no-unused-vars
-  const code = args.join(" ");
+  const { output, error } = this.tryEval(args.join(" "));
+  const cleaned = await client.clean(client, output);
+  // sends output as a file if it exceeds the character limit
+  // note: if the output is an error, it will have the ERROR heading, hence the extra 7 
+  const MAX_CHARS = error ? 3 + 2 + 7 + cleaned.length + 2 + 3 : 3 + 2 + cleaned.length + 2 + 3;
```

Comment:
this can be `const MAX_CHARS = error ? 17 + cleaned.length : cleaned.length + 10;`

Reply:
I did that so it would server as an explanation to people viewing the code, but okay.